api-reference/beta/api/connector_post_memberof.md

```diff
@@ -15,11 +15,11 @@ POST /connectors/<id>/memberOf
 | Authorization  | Bearer. Required|
 
 ### Request body
-In the request body, supply a JSON representation of [connectorGroup](../resources/connectorgroup.md) object.
+In the request body, supply a JSON representation of [connectorGroups](../resources/connectorgroup.md) object.
```

Comment:
This is weird, and maybe pervasive in our docs.  When I read this, it seems like I need to include the entire connectorGroup definition, rather than a fully quality URI/link to a connectorGroup resource.  Again this is defining a link to a connectorGroup, not connectorGroups.